openstax/settings/base.py

```diff
@@ -259,7 +259,7 @@
     ('0 2 * * *', 'django.core.management.call_command', ['delete_resource_downloads']),
     ('0 6 * * *', 'django.core.management.call_command', ['update_resource_downloads']),
     ('0 0 8 * *', 'django.core.management.call_command', ['update_schools_and_mapbox']),
-    ('0 9 * * *', 'django.core.management.call_command', ['update_opportunities']),
+    # ('0 0 1 * *', 'django.core.management.call_command', ['update_opportunities']),
```

Comment:
to mitigate risk, this is going to be run manually, not on a schedule
main priority here is to get adoption data for folks before a mass renewal email goes out